pipelines/notify-slack-on-failure-sample/notify-slack-on-failure-sample.yaml

```
@@ -0,0 +1,42 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: notify-slack-on-failure-sample
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: Sample pipeline that runs the "notify-slack-on-failure" task
+  params:
+    - name: secretName
+      type: string
+    - name: secretKeyName
+      type: string
+    - name: release
+      type: string
+  workspaces:
+    - name: release-workspace
+  tasks:
+    - name: notify-slack-on-failure
+      taskRef:
+        resolver: "git"
+        params:
+          - name: url
+            value: https://github.com/Paul123111/community-catalog.git
```

```suggestion
            value: https://github.com/konflux-ci/community-catalog.git
```
It would be better practice to use `taskGitUrl` and `taskGitRevision` as parameters at the top of the pipeline spec.  This approach would allow you to specify all the tasks repo and branch, allowing users to switch between forks or different branches, such as production, without having to modify the pipeline. It will also be needed for testing. An example of this can be seen here 
  https://github.com/konflux-ci/release-service-catalog/blob/ab7f9e593e41d28d7d968f6a10f8b2000da7a148/pipelines/managed/fbc-release/fbc-release.yaml#L50-L56

We shouldn't really be hard-coding in params

---

Yeah, I added a similar comment a few minutes later :) https://github.com/konflux-ci/community-catalog/pull/4#discussion_r2091087576

Both tenant and final pipelines will be started with taskGitUrl and taskGitRevision as parameters, so we should use those.